bionic/persistence.py

```diff
@@ -602,16 +602,6 @@ def _find_best_match(self, query):
                 level="exact",
             )
 
-        samecode_url_prefix = self._samecode_metadata_url_prefix_for_query(query)
-        samecode_urls = [
-            url for url in equivalent_urls if url.startswith(samecode_url_prefix)
-        ]
-        if len(samecode_urls) > 0:
```

Comment:
For clarity, this wasn't doing anything in the current code, right? I don't see any place in Bionic where we create a file with `samecode_url_prefix`.

Reply:
Yes, this seems to be vestigial. I actually have a separate commit for removing these unused bits, but it's not very clear from the PR message.